packages/wonder-blocks-timing/docs.md

```diff
@@ -1,11 +1,398 @@
 Abstractions for common timing APIs like `setTimeout`, `setInterval` and
-`requestAnimationFrame`.
+`requestAnimationFrame` that are aware of React component lifecycles, ensuring
+that scheduled timer actions are not unexpectedly dangling after a component
+unmounts.
 
-## Migration
+Access to the timing API is provided via the `withActionScheduler` higher order
+component.
+
+### Timing API Usage Example
+
+The following component, `MyNaughtyComponent`, will keep spamming our pretend
+log even after it was unmounted.
+
+```jsx
+const Button = require("@khanacademy/wonder-blocks-button").default;
```

Comment:
Not required but nice to have..... Now that we've updated Styleguidist, it would be great if we could use `import` here instead of `require`.

Reply:
Done